Extension/Context.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Netdudes\DataSourceryBundle\Extension;
+
+class Context
```

Comment:
With the changes made to support a context aware function, this now makes more sense. :+1: